control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -518,6 +516,14 @@ void VehicleCmdGate::onTimer()
     }
     gear_cmd_pub_->publish(gear);
   }
+
+  publishStatus();
```

Comment:
I think that the implementation have changed because, when there is an early return due to conditions like !isDataReady() or is_emergency_state_heartbeat_timeout_, not all Status messages are output. Is this not an issue?

Reply:
You are right.
Then there are 2 options:
- `publishStatus()` is called in a separate timer callback (current implementation before this PR) but its runtime is not measured (this is probably fine).
- `publishStatus()` is called before every early returns.

Reply:
> publishStatus() is called in a separate timer callback (current implementation before this PR) but its runtime is not measured (this is probably fine).

The processing within the publishStatus function is simple and lightweight, so I think this should be fine

Reply:
Done: a5a4d92b87